C#/C#.sublime-syntax

```diff
@@ -1541,7 +1546,7 @@ contexts:
 
   finally_block:
     - meta_scope: meta.block.trycatch.cs
-    - match: (finally)\b\s*
+    - match: finally\b
```

Comment:
I didn't remove it :) I only removed `\s*`, which meant that whitespace after `finally` was getting scoped as `keyword.control.trycatch.finally.cs`